server/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapper.java

```diff
@@ -496,9 +510,13 @@ public IndexFieldData.Builder getFieldDataBuilder(String name) {
 
             private static void validateParsed(float value) {
                 if (Float.isFinite(value) == false) {
-                    throw new IllegalArgumentException("[float] supports only finite values, but got [" + value + "]");
+                    throwOnInfiniteFloat(value);
```

Comment:
Same as previous comment, is this bit important?

Reply:
See https://github.com/elastic/elasticsearch/pull/85688#discussion_r857494685